jsapp/js/constants.ts

```diff
@@ -623,4 +623,10 @@ const constants = {
 
 export const HELP_ARTICLE_ANON_SUBMISSIONS_URL = 'managing_permissions.html';
 
+export enum ReactQueryStatuses {
```

Comment:
Is there a reason we can't just import `QueryStatus` from react-query wherever we need it?

Reply:
Didn't know about this :D I'll remove it in favour of `QueryStatus`